src/project-guideline/index.md

```diff
@@ -0,0 +1,14 @@
+---
+layout: layouts/default.njk
+---
+
+# Project Guideline
+## What?
+This is a list of guidelines to consider following for Zinc projects. It was initially created using thoughts/ideas from our Notabli Retrospective, but ideally will become a living document, contributed to over time.
```

Comment:
nit-pick, I don't think we need to share the history, but maybe we want to share the intent. I.e. "We've learned by doing that there are some things that make new projects flow more smoothly. This is a living document that will change over time, and was last updated on May 5th, 2020 "